### DISCUSSION TITLE
Developer Meeting (September 6 2023): Crazyflie Lib

### DISCUSSION THREAD

**Discussion by Bananazooka** [bitcraze/discussions#884] [Announcements]
The next developer meeting will be on the 6th of September, where we will talk about the Crazyflie lib, its status, architecture, and its future. 

**Date**: Wednesday, September 6 2023
**Time**: 3 PM Central European Summer Time ([check your timezone](https://www.worldtimebuddy.com/?qm=1&lid=5128581,100,2692969,12,14,1816670&h=2692969&date=2023-9-6&sln=15-16&hf=1))
**Topic**: Crazyflie lib
**Host**: @ataffanel 
**Meeting link**: the meeting is over ! Click [here](https://youtu.be/JwesV3hOFlw) to see the recording
**Agenda**:
15:00 - 15:30: Presentation and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**Comment by slivingston** [comment]
Thanks for the meeting! About having to reboot periodically, here is the Airbus bug: <https://ad.easa.europa.eu/ad/2017-0129R1>

> To address this potential unsafe condition, Airbus issued the AOT to provide instructions to reset
the internal timer. Consequently, EASA issued AD 2017-0129 to require repetitive on ground power
cycles (resets).

**Reply by krichardsson** [reply]
Thanks!

**Comment by Bananazooka** [comment]
Thanks everyone! The meeting is over but you can watch its recording on [Youtube](https://youtu.be/JwesV3hOFlw)
